### PR TITLE
Add overview table.

### PR DESCRIPTION
**Pull request by dereuromark**
Better overview

### CODE REVIEW

File: docs/manifest-types.md [hunk 1]
```diff
@@ -29,6 +29,23 @@ All manifests have such structure for common types. More data you can find below
 
 ## Available manifest types
 
+| Type                                                  | Generation | Integration |
+|-------------------------------------------------------|------------|-------------|
+| [wire-plugin](#wire-plugin)                           | YES        | YES         |
```
Comment (dereuromark) [inline docs/manifest-types.md:34]:
```suggestion
| [wire-plugin](#wire-plugin-manifest)                           | YES        | YES         |
```

I think the keys must be the slug of the title, unfortunately, for the links to work

**Comment by dereuromark** [comment]
Who can finish this up? @Sento2012 @yaroslav-spryker etc

**Comment by Sento2012** [comment]
I'll can in saturday-sunday

**Comment by dereuromark** [comment]
Any update on this?

File: docs/manifest-types.md (hunk 1 above)
Comment (VladislavStrelchenko) [reply docs/manifest-types.md]: We just have to use a full slug, for example `wire-plugin-manifest`

**Comment by VladislavStrelchenko** [comment]
@dereuromark Now is up to date

**Pull request merged**